packages/core/src/schema/models/promptVersions.ts

```diff
@@ -26,8 +26,11 @@ export const promptVersions = latitudeSchema.table('prompt_versions', {
   ...timestamps(),
 })
 
-export const promptVersionRelations = relations(commits, ({ one }) => ({
```

Comment:
This was a bug. It has to be the table that has the relations. In this case `promptVersions`.

Is very nice to use Drizzle Studio because is super strict and if we have some relation miss configured it does not run

```
Drizzle Studio is up and running on https://local.drizzle.studio?port=3002&host=0.0.0.0
Error: There is not enough information to infer relation "__latitude__.commits.commit"

```